src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import {useState} from 'react';
+
+export default function FormPage() {
+	const [invoice, setInvoice] = useState(false);
+	const [name, setName] = useState('');
+	const [amount, setAmount] = useState('');
+
+	return (
+		<div>
+			{invoice ? (
+				<form>
+					<ul>
+						<li>{name}</li>
+						<li>{amount}</li>
+					</ul>
+				</form>
+			) : (
+				<div>
+					<label htmlFor="name">Name</label>
+					<input
+						type="text"
+						id="name"
+						placeholder="John Dylen"
+						value={name}
+						onChange={() => setName(event.target.value)}
+					/>
+					<label htmlFor="amount">amount</label>
+					<input
+						type="text"
+						id="amount"
```

Comment:
same as above

Reply:
The id connects the input with the label but your suggestion should be added too. Thank you for your review!